CMakeLists.txt

```diff
@@ -92,5 +92,4 @@ if(("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR
 endif()
 
 add_subdirectory(src)
-
-install(FILES LICENSE DESTINATION .)
```

Comment:
Ok. Lets see, what it leads to in unix systems when you do the standard workflow ([`./configure && make && make install`](https://robots.thoughtbot.com/the-magic-behind-configure-make-make-install)).

Imagine I've build the project and I want to install BigARTM to the system:
```
root@0f1d2ab50075:/tmp/bigartm/build# make install
[  3%] Built target gflags-static
[  8%] Built target google-glog
[ 33%] Built target libprotobuf
[ 57%] Built target libprotoc
[ 58%] Built target protoc
[ 86%] Built target artm-static
[ 86%] Built target artm
[ 98%] Built target artm_tests
[100%] Built target cpp_client
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/./LICENSE
-- Installing: /usr/bin/protoc
-- Set runtime path of "/usr/bin/protoc" to "$ORIGIN/../lib"
-- Installing: /usr/python/artm/library.py
-- Installing: /usr/python/artm/artm_model.py
-- Installing: /usr/python/artm/messages_pb2.py
-- Installing: /usr/python/artm/__init__.py
...
```
Stop, wtf BigARTM's `LICENSE` file do in my userspace dir, common for all the applications???